history-network.md

```diff
@@ -209,7 +209,7 @@ encoded_uncles          = rlp.encode(list_of_uncle_headers)
 Note 1: The type-specific encoding might be different in future transaction types, but this encoding
 works for all current transaction types.
 
-Note 2: The `list_of_uncle_headers` refers to the array of uncle headers [defined in the devp2p spec](https://github.com/ethereum/devp2p/blob/master/caps/eth.md#block-encoding-and-validity).  
+Note 2: The `list_of_uncle_headers` refers to the list of uncle headers [defined in the Ethereum yellow paper](https://github.com/ethereum/yellowpaper/blob/30782852fef61f61a247dafbdad814ae7e00fb39/Paper.tex#L399) and is represented as an array of byte arrays where each byte array is the raw bytes representing an uncle header.  
```

Comment:
I think you rather need to say here that it is an RLP encoded list of block headers (from the uncle blocks) that results in a byte string/array. 

E.g.:
```suggestion
Note 2: The `list_of_uncle_headers` refers to the list of uncle headers [defined in the Ethereum yellow paper](https://github.com/ethereum/yellowpaper/blob/30782852fef61f61a247dafbdad814ae7e00fb39/Paper.tex#L399) and is represented RLP encoded list of block headers, where each block header is the header from an uncle block.
```

Also, I think a bit more relevant yellow paper link is this line: https://github.com/ethereum/yellowpaper/blob/30782852fef61f61a247dafbdad814ae7e00fb39/Paper.tex#L414 ?

Reply:
I'd be hesitant to say it's an RLP encoded list since that's kind of what the larger `content: rlp(list_of_uncle_headers)` already says and I'm talking specifically about what's inside the parentheses.  I wouldn't want implementers to think we mean our actual code should be `rlp.encode(rlp.encode(list_of_uncle_headers))`.  Otherwise, I'm good with the above.

Reply:
Right, forgot about that. Then I'd suggest dropping everything from the `and` onwards. I mostly wanted to point out that the byte array wording is not really correct.

Reply:
I went with @carver's suggestion.  Any last comments before we merge this?